README.md

```diff
@@ -35,8 +35,10 @@
 | [react-copy-write](https://github.com/aweary/react-copy-write) | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/react-copy-write) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/react-copy-write) |                Coming Soon - Contribute Here?                |
 | [react-simplified](https://gitlab.com/eidheim/react-simplified) | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/react-simplified) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/react-simplified) | [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/ReactNative/react-simplified) |
 | [reim](https://github.com/IniZio/reim)                       | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/reim) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/reim) |                Coming Soon - Contribute Here?                |
+| [React 16.x Context](https://reactjs.org/docs/context.html) + [Redux](https://github.com/reactjs/react-redux)                       | [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/redux-and-context) |                Coming Soon - Contribute Here?                |
 | [ReComponent](https://github.com/philipp-spiess/react-recomponent)                       | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/react-recomponent) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/react-recomponent) |                Coming Soon - Contribute Here?                |
 | [undux](https://github.com/bcherny/undux)                       | [<img src="./_art/csb.png" width="40px" />](https://codesandbox.io/s/github/GantMan/ReactStateMuseum/tree/master/React/undux) [<img src="./_art/octo.png" width="36px" />](https://github.com/GantMan/ReactStateMuseum/tree/master/React/undux) |                Coming Soon - Contribute Here?                |
+>>>>>>> a6e8515d2f7bc8daf437fd43072128ad6467a57d
```

Comment:
```suggestion

```

Looks like there's a merge artifact still in the Readme.  Best to remove.

Reply:
Whoops! Didn't catch that. Should be gone now. 😄